MeshLib/MeshInformation.h

```diff
@@ -70,6 +70,16 @@ class MeshInformation
      */
     static std::array<unsigned, 7> getNumberOfElementTypes(
         const MeshLib::Mesh& mesh);
+
+    /// writes all numbers of element types
+    static void writeAllNumbersOfElementTypes(const MeshLib::Mesh& mesh);
+
+    /// writes out property vector information
+    static void writePropertyVectorInformation(const MeshLib::Mesh& mesh);
+
+    /// writes out mesh validation results
+    // Remark: MeshValidation can modify the original mesh
```

Comment:
```suggestion
    /// Remark: MeshValidation can modify the original mesh
```
Otherwise the comment above this one goes to file-documentation and the function is not documented.